### DISCUSSION TITLE
Firmware v2 / asset creation tips

### DISCUSSION THREAD

**Discussion by marian-m12l** [marian-m12l/studio#161] [General]
Feel free to use this thread to share your tips regarding firmware v2 / asset generation or tuning, or ask for advice.

**Comment by djedevienne** [comment]
Bonjour, merci pour ce travail . Pour moi tout fonctionne presque. je suis sur une nouvelle lunii v2 et lorsque j'ajoute une histoire et que je souhaite la lancé sur la lunii j'entends le titre et après comme si c'étais un cd rayé. A ce moment la je suis obligé de redemarrer.
Vous avez une piste?

**Reply by djedevienne** [reply]
j'ai trouvé en faites c etais le format de mes annonces qui n'était pas bon!! une fois mis en mono ca fonctionne!
Par contre quand je met un ancien package ca bug, je suis obligé de réenregistrer toutes les annonces! dommage que la conversion n'agisse pas sur les menu!

**Reply by marian-m12l** [reply]
@djedevienne Merci pour les retours. La conversion s'applique à tous les fichiers audio. Est-ce qu'il n'y a pas une différence d'encodage du fichier audio "source" entre les menus et les histoires ?

**Reply by djedevienne** [reply]
si je pense, mais ce n'est pas moi qui ai fait le package!
